### ISSUE TITLE
KDSoap does not produce moc_ files when using a shadow build setup

### ISSUE THREAD

**Issue by DRFrederik** [KDAB/KDSoap#79]
When I build KDSoap (no modifications locally whatsoever) using shadow build, the project fails at the following first error:

KDSoap/kdwsdl2cpp/common/fileprovider.cpp:104: error: moc_fileprovider.cpp: No such file or directory
 #include "moc_fileprovider.cpp"

QMake was run prior to building the project. When searching for the file moc_fileprovider.cpp, it does not seems to exist anywhere

When building regularly (no shadow build) the build completes and the file moc_fileprovider.cpp is located at:
KDSoap/kdwsdl2cpp/common/.moc/moc_fileprovider.cpp

**Comment by winterz** [comment]
is this using git master? or from a released tarball?

works for me:
 mkdir build; cd build; ../configure.sh -shared -debug; make

perhaps you need to cleanout your build dir and start over?

**Comment by DRFrederik** [comment]
No, it is when using the project in QtCreator. Opening kdsoap.pro then select shadow build. The point is my customer builds every project using a shadow build dir, including kdsoap.pro as a subdir to produce a .so shared lib only works when I deselect shadow build.
